.github/workflows/packaging.yml

```diff
@@ -101,3 +101,74 @@ jobs:
           jupyter nbextension list 2>&1 | grep -ie "voila/extension.*enabled" -
           voila --version
           voila --help
+
+  release:
+    # TODO
+    # if: startsWith(github.ref, 'refs/tags/')
+    needs: [install]
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
```

Comment:
To be able to get the previous tag in the following steps with `git rev-list --tags`